src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Drivers/ContentCreatedEventDisplayDriver.cs

```diff
@@ -0,0 +1,13 @@
+using OrchardCore.ContentManagement.Metadata;
```

Comment:
Is the aim of this PR to rename or add new display driver(s)  too?

Reply:
I don't know why those ones haven't been dtected as renamed like the others.